### ISSUE TITLE
Investigate Getting Most Recent Hash By Branch

### ISSUE THREAD

**Issue by alexfinnarn** [CuBoulder/lil_shrugger#126]
Since Atlas only stores the commit hash from a branch, you can't assume that the most recent hash for a code asset that comes up in edit mode is the one for your branch. 

There could be a way to get the most recent commit of the branch you added but it would require discipline in adding, the version I guess, manually when you create an asset. 

https://api.github.com/repos/CuBoulder/express/branches can give you the most recent commit per branch and then the API gives the default branch in order to look up that hash. 

So, there could be a most recent hash of the default branch and then an attempt to look up the most recent hash of your branch.

**Comment by alexfinnarn** [comment]
I've reworked the `store.commit('addEditContent', options);` event to only include a string. Trying to make it an object loses the dependency tracking that Vue provides. Since I was only changing a property of a computed object, the component doesn't update that data. When I changed the computed variable to a string

The best place to get most recently updated branches seems to be at `https://github.com/CuBoulder/${repoName}/branches/active?page=1` and parsing the DOM for `document.querySelector('div[data-branch-name]').getAttribute('data-branch-name');`. However, due to CORS making that request is impossible. 

Adding the branch name to the code asset would solve the problem of "what branch does this commit hash belong to?" 

Using Electron would mitigate the CORS issue, and also, if the code is local, then any sort of `git` command manipulation is also possible. It is trivial to download Express, Drupal, and all the bundles to your laptop and use those code assets even if all they do is sit in a directory.

https://api.github.com/repos/CuBoulder/express/branches

The "branches" URL can give you the latest commits of all branches on a repo, but it is sorted alphabetically and has no way of tieing "last updated" to those branches.
